src/app/banner/banner.component.spec.ts

```diff
@@ -61,18 +61,15 @@ const declarations = [
   AliciaKeys
 ];
 
-const providers = [BannerService];
-
-describe('BannerComponent', () => {
+xdescribe('BannerComponent', () => {
```

Comment:
Le describe est toujours skipped ?!

Reply:
Fixed in 9848f52998b7a185df73f3b3962af750a949c55f